using_images/db_images/mongodb.adoc

```diff
@@ -64,122 +38,11 @@ This image is available on DockerHub. To download it:
 `$ docker pull openshift/mongodb-24-centos7`
 ****
 
-Alternatively, you can build a CentOS 7 based MongoDB image:
-
-. Clone the following repository:
-+
-****
-`$ git clone https://github.com/openshift/mongodb.git`
-****
-. Navigate into the directory you just created:
-+
-****
-`$ cd mongodb`
-****
-. Build the image:
-+
-****
-`$ make build VERSION=2.4`
-****
-
-[NOTE]
-====
-If you omit the `VERSION` parameter, the build/test action is performed on all
-provided versions of MongoDB. Since version `2.4` is the only verision provided,
-you can omit this parameter.
-====
-
-== Usage
-
-To create a container running MongoDB, you need to run following commands
-depending on your base image.
-
-*RHEL 7 Base Image*
-
-To create a container running MongoDB with a RHEL 7 base image:
-
-****
-`$ docker run -d -e MONGODB_USERNAME=_<user>_ -e MONGODB_PASSWORD=_<password>_ -e MONGODB_DATABASE=_<database>_ -p 27017:27017 openshift/mongodb-24-rhel7`
-****
-
-This creates a container named with the credentials and database that you
-specified.
-
-*CentOS 7 Base Image*
-
-To create a container running MongoDB with a CentOS-7 base image:
-
-****
-`$ docker run -d -e MONGODB_USERNAME=_<user>_ -e MONGODB_PASSWORD=_<password>_ -e MONGODB_DATABASE=_<database>_  -p 27017:27017 openshift/mongodb-24-centos7`
-****
-
-To access and test if database is running:
-
-****
-`$ mongo 127.0.0.1/_<database>_ --eval "db"`
-****
-
-To properly stop the container:
-
-****
-`$ docker stop _<mongodb_container>_`
-****
-
-== Testing
-
-This repository also provides a test framework which checks basic functionality
-of the MongoDB image.
-
-Select whether the MongoDB image you are testing is RHEL based or CentOS based.
-
-*RHEL 7 Base Image*
-
-To test a RHEL 7 based MongoDB image, run the test on a properly subscribed RHEL
-machine.
-
-. Navigate into the *_mongodb_* directory:
-+
-****
-`$ cd mongodb`
-****
-. Test the image:
-+
-****
-`$ make test TARGET=rhel7 VERSION=2.4`
-****
-
-*CentOS 7 Base Image*
-
-To test a CentOS 7 based MongoDB image:
-
-. Navigate into the *_mongodb_* directory:
-+
-****
-`$ cd mongodb`
-****
-. Test the image:
-****
-`$ make test VERSION=2.4`
-****
-
-[NOTE]
-====
-If you omit the `VERSION` parameter, the build/test action is performed on all
-provided versions of MongoDB. Since version `2.4` is the only verision provided,
-you can omit this parameter.
-====
-
-== Shared Volumes
+To use these images, you can either access them directly in those registries or push them into your OpenShift docker registry.  In addition, it is recommended that you create an ImageStream that points to the image, either in your docker registry or at the external location.  Your OpenShift resources can then reference the ImageStream.  You can find example ImageStream definitions for all the provided OpenShift images https://github.com/openshift/origin/tree/master/examples/image-streams[here].
```

Comment:
you mean provide the json? because then we have to maintain the json in yet another location if it changes.  centralizing and linking is more maintainable.